src/server/scripts/Northrend/Ulduar/Ulduar/boss_yoggsaron.cpp

```diff
@@ -296,6 +298,26 @@ enum Misc
     DATA_GET_DRIVE_ME_CRAZY             = 4,
 };
 
+struct LocationsXY
+{
+    float x, y, z, o;
+    uint32 id;
```

Comment:
why `uint32`? also where is this `id` used?

Reply:
oops 🐱